website/docs/docs/build/validation.md

```diff
@@ -15,13 +15,16 @@ The code that handles validation [can be found here](https://github.com/dbt-labs
 
 ## Validations command
 
-You can run validations from dbt Cloud or the command line with the following [MetricFlow commands](/docs/build/metricflow-commands):
+You can run validations against the defined semantic model configurations from the command line with the following [MetricFlow commands](/docs/build/metricflow-commands):
+
+Note, in dbt Cloud you don't need to validate the Semantic Layer config separately. Running a dbt command (such as dbt parse, dbt build, dbt compile, dbt run) automatically checks it.
 
 ```bash
-dbt sl validate-configs # dbt Cloud users
 mf validate-configs # dbt Core users
 ```
 
+You can run semantic validations in a CI job to guarantee any code changes made to dbt models don't break these metrics. For more information, refer to [semantic validation in CI](/docs/deploy/ci-jobs#semantic-validations-in-ci).
```

Comment:
> (/docs/deploy/ci-jobs#semantic-validations-in-ci)

I can't seem to click into this? Leads to nothing for me 🤔

Reply:
hey @WilliamDee sorry about that! what about now: https://docs-getdbt-6ydzf4xcw-dbt-labs.vercel.app/docs/build/validation

Reply:
ah ok, so i think [this](https://docs-getdbt-6ydzf4xcw-dbt-labs.vercel.app/docs/deploy/ci-jobs#semantic-validations-in-ci) is DWH validation, not semantic validation. Semantic validation is done through the `dbt parsing` commands

Reply:
oh got it! sorry, i'll move this to the data platform section!

Reply:
do i need to change the header in the CI job page? i thought the official name was 'semantic validations in CI'? 👀

Reply:
thanks so much @WilliamDee ! ok i've moved the semantic validations in ci to the 'data platform' section here: https://docs-getdbt-7yyzg2407-dbt-labs.vercel.app/docs/build/validation#data-platform

is that looking ok to you?

![Screenshot 2024-08-08 at 09 34 15](https://github.com/user-attachments/assets/0a6fb425-ebf9-4886-9d63-ff6f907b06e0)